js/TimePickerComponent.js

```diff
@@ -0,0 +1,19 @@
+import '../libs/jt.timepicker/jquery.timepicker.min';
+
+class TimePickerComponent {
+
+    /**
+     * Bind timepicker to element
+     * @param {jQuery} $element - jQuery object of the element that requires a time picker
+     * @param {Object} options - Options for the date picker, see https://github.com/jonthornton/jquery-timepicker
+     */
+    bindTimePicker ($element, options) {
+        if (typeof $element === 'undefined' || !$element) {
```

Comment:
Probably worth noting here that an empty jQuery collection will pass this validation. This might be the preferred behaviour, as we wouldn't want our plugin to throw necessarily in the event there isn't an `$element` on the page, however worth considering as part of this method.